libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardTiger.spec.ts

```diff
@@ -2,50 +2,107 @@
+            cy.login();
             Navigation.visit("dashboard/dashboard-tiger");
         });
 
         it("should render topBar", () => {
-            const dashboard = new Dashboard();
-
             dashboard.topBarExist();
         });
 
         // eslint-disable-next-line jest/no-disabled-tests
         it.skip("should render title", () => {
-            const topBar = new TopBar();
-
             topBar.dashboardTitleExist().dashboardTitleHasValue("Test dashboard");
         });
 
         it("should render edit button", () => {
-            const topBar = new TopBar();
-            const dashboard = new Dashboard();
-
             dashboard.topBarExist();
             topBar.editButtonIsVisible(true);
         });
 
         it("should menu button render", () => {
-            const topBar = new TopBar();
-
             topBar.menuButtonIsVisible();
         });
 
         // eslint-disable-next-line jest/no-disabled-tests
         it.skip("should open menu button and contain items", () => {
-            const topBar = new TopBar();
-
             topBar
                 .menuButtonIsVisible()
                 .clickMenuButton()
                 .topBarMenuItemExist(".s-export_to_pdf")
                 .topBarMenuItemExist(".s-schedule_emailing");
         });
+
+        //Cover ticket: RAIL-4747
+        it(
+            "Should display delete button on edit mode and hidden on view mode",
+            { tags: ["checklist_integrated_tiger"] },
+            () => {
+                editMode.edit();
+                topBar.menuButtonIsVisible().clickMenuButton().hasDeleteButton(true);
+                dashboard.moveWidget(0, 1, WidgetDropZone.NEXT);
+                editMode.save(true);
+                topBar.menuButtonIsVisible().clickMenuButton().hasDeleteButton(false);
+            },
+        );
+
+        //Cover ticket: RAIL-4702
+        it("Should enable Save button when resize column", { tags: ["checklist_integrated_tiger"] }, () => {
+            Navigation.visit("dashboard/stage-name");
+            editMode.edit();
+            const table = new Table(".s-dash-item");
+            table.waitLoaded().resizeColumn(0, 1, 500, true);
+            editMode.saveButtonEnabled(true);
+        });
+
+        //Cover ticket: RAIL-4728
+        it(
+            "Should reload widget after check/uncheck attribute filter",
+            { tags: ["checklist_integrated_tiger"] },
+            () => {
+                const widgetConfig = new WidgetConfiguration(0);
+
+                Navigation.visit("dashboard/stage-name");
+                editMode.edit();
+                widget.waitTableLoaded().click();
+                widgetConfig.openConfiguration().checkFilterCheckbox(STAGE_NAME_CHECKBOX);
+                widget.isLoading(true);
+                widgetConfig.checkFilterCheckbox(STAGE_NAME_CHECKBOX);
+                widget.isLoading(true);
+            },
+        );
+
+        //Cover ticket: RAIL-4772
+        it(
```

Comment:
libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboard.spec.ts
this test is duplicated with test in above link. Why don't you put them in a same place, use { tags: ["checklist_integrated_tiger", "checklist_integrated_bear"] },